.github/workflows/docs.yaml

```diff
@@ -0,0 +1,67 @@
+---
+"name": "Docs (release notes)"
+"on":
+  "push":
+  "pull_request":
+  "workflow_dispatch":
```

Comment:
Do we want to run it always or is it enough to run it when the imagestream yaml file is updated only since we use just these to read the data?

Reply:
It's very fast, so I vote for running always.